api/repository.go

```diff
@@ -34,12 +34,17 @@ type Repository struct {
 	// If empty, then Bytebase won't auto generate it.
 	SchemaPathTemplate string `jsonapi:"attr,schemaPathTemplate"`
 	// The file path template for matching the sql files for sheet.
-	SheetPathTemplate  string `jsonapi:"attr,sheetPathTemplate"`
-	ExternalID         string `jsonapi:"attr,externalId"`
-	ExternalWebhookID  string
-	WebhookURLHost     string
-	WebhookEndpointID  string
-	WebhookSecretToken string
+	SheetPathTemplate string `jsonapi:"attr,sheetPathTemplate"`
+	// If true, we will create a pull request for repository to setup the SQL review CI.
+	EnableSQLReviewCI bool `jsonapi:"attr,enableSQLReviewCI"`
+	// SQLReviewCIPullRequestURL is the pull request URL to setup the SQL review CI.
+	// Only return this value if we need to auto-create the pull request for users.
+	SQLReviewCIPullRequestURL string `jsonapi:"attr,sqlReviewCIPullRequestURL"`
```

Comment:
Do we need this or persist the CI setup PR?

Reply:
We don't need to persist this but need it for the frontend -> https://github.com/bytebase/bytebase/pull/2981/files#diff-36b04cb3c7a39449e0b8d270fe126a21cea10e9b2f0e4f68a76c3bff344533f4R259
The frontend needs to URL so that we can redirect users to the pull request.

Update the comment here.

Reply:
SG.